roofit/roofitcore/test/testRooFuncWrapper.cxx

```diff
@@ -0,0 +1,52 @@
+#include <RooRealVar.h>
+#include <RooWorkspace.h>
+#include <RooRealProxy.h>
+#include <RooGaussian.h>
+#include <RooFuncWrapper.h>
+#include <RooDataSet.h>
+#include <RooAbsPdf.h>
+#include <TROOT.h>
+#include <TSystem.h>
+#include <RooFitResult.h>
+#include <RooListProxy.h>
+#include <TMath.h>
+
+#include "gtest/gtest.h"
+
+class Interface : public ::testing::Test {};
```

Comment:
```suggestion
```
You don't need this extra base class if your test is not parametrized or a fixture.